pkgs/development/compilers/ghc/7.10.2.nix

```diff
@@ -34,7 +34,7 @@ stdenv.mkDerivation rec {
     sha256 = "1x8m4rp2v7ydnrz6z9g8x7z3x3d3pxhv2pixy7i7hkbqbdsp7kal";
   };
 
-  buildInputs = [ ghc perl libxml2 libxslt docbook_xsl docbook_xml_dtd_45 docbook_xml_dtd_42 hscolour ];
+  buildInputs = [ ghc perl libxml2 libxslt docbook_xsl docbook_xml_dtd_45 docbook_xml_dtd_42 hscolour.bin ];
```

Comment:
All those accesses to `.bin` should be replaced by use of `getBin`. These variables are function arguments, so we really don't know what kind of derivation will be passed there and cannot rely on the fact that the `bin` output actually exists. People might call this derivation passing `hscolour.bin` for all we know.